### DISCUSSION TITLE
Bug? Cloned animal log

### DISCUSSION THREAD

**Discussion by Pletoo** [sheltermanager/asm3#1529] [General]
When an animal record is cloned, the first history log is copied too.
I created "Hickory" then cloned him and renamed the copy "Hugs."
On both records the History log says:

> Hickory: Moved from -* to Shelter

We don't have any other movements so I don't look at the history log often. I just happened to notice it today and didn't know if it was relevant or impacting reports.

**Comment by bobintetley** [comment]
Logs are cloned with the rest of the animal record by default. There is a hidden system option to disable log cloning if you don't want it. It can be done by executing this command at Settings->SQL Interface:

INSERT INTO configuration ('CloneAnimalIncludeLogs', 'No');
